doc/userguide/rules/app-layer.rst

```diff
@@ -8,18 +8,31 @@ Match on the detected app-layer protocol.
 
 Syntax::
 
-    app-layer-protocol:[!]<protocol>;
+    app-layer-protocol:[!]<protocol>(,<mode>);
 
 Examples::
 
     app-layer-protocol:ssh;
     app-layer-protocol:!tls;
     app-layer-protocol:failed;
+    app-layer-protocol:!http,final;
+    app-layer-protocol:http,toserver; app-layer-protocol:tls,toclient;
+    app-layer-protocol:http2,final; app-layer-protocol:http1,orig;
```

Comment:
mismatch with doc below `orig` vs `original`

Reply:
Nice catch, fixing the doc